app/lib/audit.js

```diff
@@ -153,9 +162,24 @@ function validate_record(r, k, ln){
 }
 
 var logs = {},
-    file = fs.createWriteStream('/opt/data/audit_log.tsv');
+    file = fs.createWriteStream('/opt/data/audit_log.tsv'),
+    file_sql = fs.createWriteStream('/opt/data/audit_log.sql');
+
+file_sql.write("INSERT INTO mdl_log (time,userid,ip,course,module,cmid,action,url,info) VALUES ");//the first part of the INSERT command will be written only once, to make execution faster
 
 function audit(table, module, action){
+    fs.readFile('/opt/data/audit_log.sql', function (err,data) {//needed to add a ";"" at the end of the sql commands
```

Comment:
This seems wrong... Surely we shouldn't read+write a file in passing every time we access append() or flush()?